frontend/src/pages/chat/components/chat-footer/styles.module.scss

```diff
@@ -0,0 +1,28 @@
+.chat-footer {
+  position: fixed;
+  bottom: 0;
+  display: flex;
+  align-items: center;
+  width: calc(70% - 88px);
```

Comment:
@Liza-Veis  As I undestand we should avoid using relative values for layout and fixed values for width and height. So we can't also use such constractions?